db/kvdb/kvdb_test.go

```diff
@@ -220,8 +220,8 @@ func TestConcurrentDeleteOldCheckpoints(t *testing.T) {
 		err = db.MakeCheckpoint()
 		require.NoError(t, err)
 		go func() {
-			err = db.DeleteOldCheckpoints()
-			require.NoError(t, err)
+			err2 := db.DeleteOldCheckpoints()
```

Comment:
try to use a more intuitive name
```suggestion
			errDelete := db.DeleteOldCheckpoints()
```

Reply:
also, only adding the `:` works too, because you are going to assign a new variable for this small scope

 `err := db.DeleteOldCheckpoints()`